### PR TITLE
Javadocs with return statements

### PR DESCRIPTION
**Pull request by allisonport-db**
• Regenerated javadocs with `@return` statements
• Updated `build.sbt` to not disable `@return` tags
• Explicitly bump up `genjavadoc` version to 0.18 to work with Scala 2.12 (it's done in Delta OSS figured it won't hurt)

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -495,7 +499,6 @@ lazy val standalone = (project in file("standalone"))
       "-public",
       "-windowtitle", "Delta Standalone Reader " + version.value.replaceAll("-SNAPSHOT", "") + " JavaDoc",
       "-noqualifier", "java.lang",
-      "-tag", "return:X",
```
Comment (scottsand-db) [inline build.sbt:498]: nice catch!

File: build.sbt [hunk 2]
```diff
@@ -427,7 +427,11 @@ lazy val standalone = (project in file("standalone"))
       ),
       "org.scalatest" %% "scalatest" % scalaTestVersion % "test",
       "org.slf4j" % "slf4j-api" % "1.7.25",
-      "org.slf4j" % "slf4j-log4j12" % "1.7.25"
+      "org.slf4j" % "slf4j-log4j12" % "1.7.25",
+
+      // Compiler plugins
+      // -- Bump up the genjavadoc version explicitly to 0.18 to work with Scala 2.12
+      compilerPlugin("com.typesafe.genjavadoc" %% "genjavadoc-plugin" % "0.18" cross CrossVersion.full)
```
Comment (scottsand-db) [inline build.sbt:434]: was this not working before? i.e. what happens if we don't have this? just curious

**Review by scottsand-db — APPROVED** [review]
LGTM

**Review by zsxwing — APPROVED** [review]
LGTM

**Pull request merged**